test/mlir/accelerators/nnpa/driver/ccfd.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: ccfd=$(dirname %s)/ccfd.onnx && curl -L https://github.com/IBM/ai-on-z-fraud-detection/raw/main/onnx%20models/ccf_lstm_static_tf2onnx_OS_new.onnx -o ${ccfd} && onnx-mlir --mcpu=z16 --maccel=NNPA --EmitMLIR --printIR -tag="test" ${ccfd} | FileCheck %s && rm -rf ${ccfd}
+// RUN: ccfd=$(dirname %s)/ccfd.onnx && curl -L https://github.com/IBM/ai-on-z-fraud-detection/raw/main/onnx%20models/ccf_lstm_static_tf2onnx_OS_new.onnx -o ${ccfd} && onnx-mlir --mcpu=z16 --maccel=NNPA --enable-compiler-stick-unstick=false --EmitMLIR --printIR -tag="test" ${ccfd} | FileCheck %s && rm -rf ${ccfd}
```

Comment:
I think you have to set `--enable-compiler-stick-unstick` to false because `--EmitMLIR` includes generating compiler stick/unstick. If we can emit an IR before generating compiler stick/unstick, we don't have to do that. `--EmitZLowIR` is before `--EmitMLIR`, but `--EmitZLowIR` does not include memref normalization and optimizations to product the optimized code here. Perhaps, we can introduce a new --EmitXXXIR for the IR just before converting to affine. Anyway, it would be for another PR.

Reply:
So what do you suggest for now?

Reply:
Nothing to do at this moment.